docs/README.md

```diff
@@ -72,10 +250,17 @@ Note:
   verified in the `Extension` window `Runtime Status` tab for `Ansible`
   extension.
 
-## Features
+## **Features**
```

Comment:
We can remove the features section completely, since this information is duplicated under the Ansible Language Server section. Though before removal, we should make sure the information is up to date with this section. 

Instead of having this here, there could be a note in one of the docs pages about ALS, that links to that doc section.